reference/7/Microsoft.PowerShell.Core/About/about_numeric_literals.md

```diff
@@ -155,53 +157,58 @@ PowerShell supports the following type accelerators:
 > The following type accelerators were added in PowerShell 6.2: `[short]`,
 > `[ushort]`, `[uint]`, `[ulong]`.
 
-### Working with other numeric types
-
-To work with any other numeric types you must use type accelerators, which is
-not without some problems. For example, high integer values are always parsed
-as double before being cast to any other type.
-
-```
-PS> [bigint]111111111111111111111111111111111111111111111111111111
-111111111111111100905595216014112456735339620444667904
-```
-
-The value is parsed as a double first, losing precision in the higher ranges.
-To avoid this problem, enter values as strings and then convert them:
-
-```
-PS> [bigint]'111111111111111111111111111111111111111111111111111111'
-111111111111111111111111111111111111111111111111111111
-```
-
 ## Examples
 
 The following table contains several examples of numeric literals and lists
 their type and value:
 
-|  Number  |  Type   |    Value     |
-| -------: | ------- | -----------: |
-|      100 | Int32   |          100 |
-|     100D | Decimal |          100 |
-|     100l | Int64   |          100 |
-|    100uL | UInt64  |          100 |
-|    100us | UInt16  |          100 |
-|    100uy | Byte    |          100 |
-|     100y | SByte   |          100 |
-|      1e2 | Double  |          100 |
-|     1.e2 | Double  |          100 |
-|    0x1e2 | Int32   |          482 |
-|   0x1e2L | Int64   |          482 |
-|   0x1e2D | Int32   |         7725 |
-|     482D | Decimal |          482 |
-|    482gb | Int64   | 517543559168 |
-| 0x1e2lgb | Int64   | 517543559168 |
+|  Number   |  Type   |    Value     |
+| ---------:| ------- | -----------: |
+|       100 | Int32   |          100 |
+|      100D | Decimal |          100 |
+|      100l | Int64   |          100 |
+|     100uL | UInt64  |          100 |
+|     100us | UInt16  |          100 |
+|     100uy | Byte    |          100 |
+|      100y | SByte   |          100 |
+|       1e2 | Double  |          100 |
+|      1.e2 | Double  |          100 |
+|     0x1e2 | Int32   |          482 |
+|    0x1e2L | Int64   |          482 |
+|    0x1e2D | Int32   |         7725 |
+|      482D | Decimal |          482 |
+|     482gb | Int64   | 517543559168 |
+|  0x1e2lgb | Int64   | 517543559168 |
+| 0b1011011 | Int32   |           91 |
```

Comment:
Examples should be added here (and it should be noted somewhere else as well, if we haven't already somewhere below) for how adding the `u` suffix can alter the behaviour of hexadecimal or binary numerals.

Example:

```ps
PS> 0xFFFFFFFF
-1
PS> 0xFFFFFFFFu
4294967295 # [uint]::MaxValue
```

Similar things will happen with binary in 7.

Reply:
Done.